micro-deps/micro-deps/src/main/groovy/com/ofg/infrastructure/discovery/ServicePath.java

```diff
@@ -0,0 +1,35 @@
+package com.ofg.infrastructure.discovery;
+
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+import org.apache.commons.lang.builder.ReflectionToStringBuilder;
+
+/**
+ * Path to dependency as registered in service resolver, like ZooKeeper
+ */
+public class ServicePath {
+    private final String path;
+
+    public ServicePath(String path) {
+        this.path = path;
+    }
+
+    public String getPath() {
+        return path;
+    }
+
+    @Override
+    public String toString() {
+        return ReflectionToStringBuilder.toString(this);
```

Comment:
Do we really need all that reflection here? It's just one field, why not be explicit what we show/compare?